app/views/layouts/_header.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="container-fluid">
     <!-- Brand and toggle get grouped for better mobile display -->
     <div class="navbar-header">
-      <a class="navbar-brand" href="#">OSRA Support System</a>
+      <a class="navbar-brand" href="/">OSRA Support System</a>
```

Comment:
It points to the root which in our case loads the AA OSRA dashboard.
I thing it's more of a convention to link the header logo to the homepage. Every public website does this.